sys/include/net/zep.h

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_zep  ZEP header definitions
+ * @ingroup     net
+ * @brief
+ * @{
+ *
+ * @file
+ * @brief
+ *
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ */
+#ifndef ZEP_H
+#define ZEP_H
+
+#include <stdint.h>
+
+#include "byteorder.h"
+#include "net/ntp_packet.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define ZEP_PORT_DEFAULT        (17754) /**< default ZEP port */
+
+/**
+ * @brief   Type == Data for ZEPv2 header
+ */
+#define ZEP_V2_TYPE_DATA        (1)
+
+/**
+ * @brief   Type == Ack for ZEPv2 header
+ */
+#define ZEP_V2_TYPE_ACK         (2)
+
+/**
+ * @brief   Mask for length field
+ */
+#define ZEP_LENGTH_MASK         (0x7f)
+
+/**
+ * @brief   ZEP header definition
+ */
+typedef struct __attribute__((packed)) {
+    char preamble[2];       /**< Preamble code (must be "EX") */
+    uint8_t version;        /**< Protocol Version (must be 1 or 2) */
+} zep_hdr_t;
+
+/**
+ * @brief   ZEPv1 header definition
```

Comment:
Now I understand that ZEPv1 vs ZEPv2 is not only a question of the number of iterations in RIOT. Now that v2 is implemented, shouldn't we remove definitions for v1 to avoid confusion?